### PR TITLE
Acs 3568 add support for acs 7.3 jdk17

### PR DESCRIPTION
**Pull request by kmagdziarz**

### CODE REVIEW

File: docs/whats-new.md [hunk 1]
```diff
@@ -67,3 +67,10 @@ Alfresco SDK 4.1 provides support for Alfresco 6.2.x.
 ## Support for Alfresco 7.0.x, 7.1.x, and 7.2.x
 
 Alfresco SDK 4.4 provides support for Alfresco 7.0.x, 7.1.x, and 7.2.x.
+
+## Support for Java 17
+[Java 17](https://openjdk.java.net/projects/jdk/17/) is the latest stable long term (8 year support) version after Java 11. Alfresco 7.3.x already offers
+support for this version of the Java platform.
+
+Alfresco SDK 4.5 has been modified to add support for Java 17 as well. This way, if you're working as a developer in customisations for Alfresco 7.3.x you must
```
Comment (mstrankowski) [inline docs/whats-new.md:75]: We still compile sdk in java 11 though, so java 17 "can" be used by someone else when adding their extensions, not "must"

File: pom.xml [hunk 2]
```diff
@@ -56,17 +56,17 @@
...
+        <alfresco.share.docker.version>7.3.0-A20</alfresco.share.docker.version>
         <!--
              The following value is now obtained by looking at the
              - alfresco-community-share.version (eg. https://github.com/Alfresco/acs-community-packaging/blob/7.1.0/pom.xml#L17)
              or
              - alfresco-enterprise-share.version (eg. https://github.com/Alfresco/acs-packaging/blob/7.1.0.1/pom.xml#L18)
              for the GitHub Tag related to the above Docker version.
          -->
-        <alfresco.share.version>14.96</alfresco.share.version>
+        <alfresco.share.version>17.96</alfresco.share.version>
```
Comment (mstrankowski) [inline pom.xml:69]: The version of image (A20) relates to share in version 17.94. Should we use the newer one, or should we stick to version from image?

File: pom.xml [hunk 3]
```diff
@@ -179,6 +179,32 @@
...
         </profile>
 
+        <!-- 7.3 -->
+        <profile>
+            <id>community-73-tests</id>
+            <properties>
+                <alfresco.bomDependency.artifactId>acs-community-packaging</alfresco.bomDependency.artifactId>
+                <alfresco.platform.version>7.3.0-A20</alfresco.platform.version>
+                <alfresco.share.docker.version>7.3.0-A20</alfresco.share.docker.version>
+                <alfresco.share.version>17.96</alfresco.share.version>
```
Comment (mstrankowski) [inline pom.xml:189]: same as above, maybe we should use 17.94 instead

File: .travis.yml [hunk 4]
```diff
@@ -29,38 +36,78 @@ env:
 
 jobs:
   include:
-  
-    - name: "current version"
+    - name: "build application"
```
Comment (mstrankowski) [inline .travis.yml:39]: That's a new job, is it needed?

File: .travis.yml [hunk 5]
```diff
@@ -15,9 +20,11 @@ branches:
     - master
     - /feature\/.*/
     - /fix\/.*/
-    - sdk-4.4
+    - sdk-4.5
+    - ACS-3568-Add_support_for_ACS_7.3_jdk17
```
Comment (mstrankowski) [inline .travis.yml:24]: That line needs to be removed before merge

File: .travis.yml [hunk 6]
```diff
@@ -1,10 +1,15 @@
 import:
   - source: Alfresco/alfresco-build-tools:.travis.docker_login.yml@v1.1.5
   - source: Alfresco/alfresco-build-tools:.travis.docker_hub_login.yml@v1.1.5
-  - source: Alfresco/alfresco-build-tools:.travis.java.yml@v1.1.5
+  - source: Alfresco/alfresco-build-tools:.travis.maven_config.yml@v1.1.5
```
Comment (mstrankowski) [inline .travis.yml:4]: Please explain this change

File: pom.xml (hunk 3 above)
Comment (kmagdziarz) [reply pom.xml:189]: For SDK 4.5 we will use the final version of 7.3.0 Share, so that will be changed later.

**Review by mstrankowski — APPROVED** [review]

**Pull request merged**